subprojects/tooling-api-builders/src/main/java/org/gradle/tooling/internal/provider/runner/ClientForwardingProjectConfigurationOperationListener.java

```diff
@@ -58,13 +59,13 @@
 class ClientForwardingProjectConfigurationOperationListener extends SubtreeFilteringBuildOperationListener<ConfigureProjectBuildOperationType.Details> {
 
     private final Map<OperationIdentifier, ProjectConfigurationResult> results = new ConcurrentHashMap<>();
-    private final BuildOperationParentTracker parentTracker;
+    private final BuildOperationAncestryTracker ancestryTracker;
     private final PluginApplicationTracker pluginApplicationTracker;
 
     ClientForwardingProjectConfigurationOperationListener(ProgressEventConsumer eventConsumer, BuildEventSubscriptions clientSubscriptions, BuildOperationListener delegate,
-                                                          BuildOperationParentTracker parentTracker, PluginApplicationTracker pluginApplicationTracker) {
+                                                          BuildOperationAncestryTracker ancestryTracker, PluginApplicationTracker pluginApplicationTracker) {
```

Comment:
Nit: Strange newline. Can you add each parameter on a single line?

Reply:
I can fix this, but I was intentionally avoiding changing the formatting in places where I didn't have to.

Reply:
Fine with me.